packages/orchestration/src/util/tools.ts

```diff
@@ -0,0 +1,41 @@
+import { zodToJsonSchema } from 'zod-to-json-schema';
```

Comment:
[pp] Thanks for adding this file. Any idea how to reuse those things in Sample-Code and LangChain? Can we define those tools in somewhere else? If we reuse this, maybe also not type the tool with `ChatCompletionTool`.